### DISCUSSION TITLE
Disappearing messages in Discussions?

### DISCUSSION THREAD

**Discussion by Torinde** [86Box/86Box#3237] [General]
Had a user recently deleted his profile or something? I think many comments and replies are missing - the person was active and I think he provided interesting info, so it's a shame all that is gone. Can that be restored somehow (maybe with a "ghost" instead of name/picture? For example:

Discussion for "Add Soundblaster Live! support"
- should have 3 comments and 10 replies, but actually has only 2 comments and 1 reply?? (see first screenshot)
- last reply should be from myself, but it's not shown (see second screenshot)
![image](https://user-images.githubusercontent.com/97228894/230068426-b59d862f-0196-4114-a199-015318bea114.png)

![image](https://user-images.githubusercontent.com/97228894/230067968-dd0c0861-73d1-4e69-99e5-48d5ba089663.png)

Discussion for "Aureal Vortex sound cards emulation"
- should have 6 comments and 6 replies, but has less
- I remember there was a reply to Cacodemon345 Feb 2 "I bet he's tight-lipped by contract.", but it's not shown
![image](https://user-images.githubusercontent.com/97228894/230068905-fdf73953-5b6c-4c11-a9c8-35c67ad6041b.png)

PS. I'm not linking to those in order not to break them further.

**Comment by jriwanek** [comment]
To my knowledge, we've not removed any comments from the discussions, and we also have no control over how github handles removed messages.

**Reply by Torinde** [reply]
Thanks for replying and OK, I understand. Still a pity. Do you know a place at github where I can submit a "issue" for that? It's not good when a user deletes his profile to happen that even messages NOT written by him disappear (e.g. any message _below_ his messages).

**Reply by jriwanek** [reply]
I don't, but I suspect under the various "right to be forgotten" and GDPR and similar laws, that github don't get a choice anyways.

**Reply by Torinde** [reply]
In Issues/PRs I have seen messages from https://github.com/ghost - "I take the place of user accounts that have been deleted.", but maybe in Discussions that's not implemented :(
Or there are different methods of deleting the account and the user here choose the stricter method.

**Reply by jriwanek** [reply]
You'd really have to ask Github, not us, closing discussion as it's irrelovent to 86box, as covered, there's nothing we can do about it.

**Reply by Torinde** [reply]
I think [this](https://github.com/orgs/community/discussions/156299) is what happened also here. So, the missing comments are still there, but invisible - until the user contacts Github to restore visibility of his account.
